Formula/idris2.rb

```diff
@@ -13,7 +13,7 @@ class Idris2 < Formula
     sha256 cellar: :any, mojave:   "ab173b9e9ef6083e6ae2fe22f8a55892e80c064ec9b9838afa5b6488ae9a8529"
   end
 
-  depends_on "coreutils" => :build
+  depends_on "coreutils"
```

Comment:
The dependencies need to be listed in alphabetical order.

When you fix this, please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings. Refer to the [documentation](https://docs.brew.sh/Formula-Cookbook#commit) for more details.

Also, squash your commits together when you're done. Thanks!